implementer-guide/configuration/security.md

```diff
@@ -3,29 +3,67 @@ title: Security
+Enables/Disables the rate limiter. By default, the value is set to `enabled`.
 
 ### Window
 
-Specifies the time window in milliseconds that takes in the requests. It must be set to a value greater than 0. By default the value is set to 1000 milliseconds.
+Specifies the time window in milliseconds that takes in the requests. It must be set to a value greater than 0. By default, the value is set to `1000 milliseconds`.
 
 ### Permit Limit
 
-The maximum permitted number of requests that can be allowed in a time window. It must be set to a value greater than 0. By default the value is set to 1.
\ No newline at end of file
+The maximum permitted number of requests that can be allowed in a time window. It must be set to a value greater than 0. By default, the value is set to `1`.
+
+## Encryption
+
+Introduced with 2024 Q4 (10.3.24.1112).
+
+The sensitive assets are stored in the [Report Server Storage]({%slug storage-settings%}) in encrypted form. Now the encryption functionality is leveraged to use industry-standard encryption routines. The encryption algorithms use a pair of keys, __Main Key__ and __Backup Key__ created during the initial configuration of the Report Server:
+
+![The Encryption page of the Report Server Configuration with the buttons to download the encryption keys.](../../images/report-server-images/security-configure-encryption.png)
+
+To enable the enhanced encryption functionality, the administrator must download both _Main_ and _Backup_ private keys and type `Confirm` (case insensitive) in the input field just above the __Complete__ button to validate that the keys are safely stored. We strongly recommend storing the encryption private keys securely, for example, in a key vault.
+
+When the button _Complete_ is pressed, the existing sensitive assets will be securely encrypted and the encryption keys will be stored as environment variables for the user, who is used to run the Report Server and Report Server ServiceAgent.
+
+The administrator can generate a new pair of encryption keys through the `RESET ENCRYPTION KEYS` or upload a specific pair of encryption keys through the `OVERWRITE ENCRYPTION KEYS` buttons in the __Configuration__ page / __Security__ tab:
+
+![Buttons to reset or upload the encryption keys in the Report Server Configuration page.](../../images/report-server-images/security-reset-upload-encryption-keys.png)
+
+* __Reset Encryption Keys__ may be used when the keys need to be changed periodically for security purposes, or in case of an information leak. In this scenario, the Report Server will decrypt the sensitive assets using the existing pair of keys, generate new ones, and encrypt the assets again.
+* __Overwrite Encryption Keys__ may be used when the Report Serve or Report Serve ServiceAgent is migrated to a new machine where the encryption keys are not present. This functionality registers the keys in the environment variables and encrypts the sensitive assets with them.
```

Comment:
For the Overwrite Encryption Keys we may extend the info to say this should also be used when provisioning multiple instances of the scheduling service, besides the migration scenario. Not sure for the correct wording, as I do not find relevant article. If such exists, cross-linking would be helpful.